### PR TITLE
Mgiparser

### PR DESCRIPTION
**Pull request by pnrobinson**

### CODE REVIEW

File: phenol-core/src/main/java/org/monarchinitiative/phenol/formats/mpo/MpAllelicComposition.java [hunk 1]
```diff
@@ -0,0 +1,85 @@
+package org.monarchinitiative.phenol.formats.mpo;
+
+import java.util.Objects;
+
+/**
+ * This class represents the Allelic Composition field of the MGI_PhenoGenoMP.rpt file as a string (without breaking
```
Comment (hannahblau) [inline phenol-core/src/main/java/org/monarchinitiative/phenol/formats/mpo/MpAllelicComposition.java]: MGI_PhenoGenoMP.rpt is not the only file that has an Allelic Composition field, maybe reword this description to be more general?

Comment (pnrobinson) [reply phenol-core/src/main/java/org/monarchinitiative/phenol/formats/mpo/MpAllelicComposition.java]: I think MGI_GenePheno.rpt also has this field. But please do not accept this PR for right now, I think we need a few more tests and the documentation needs to be improved a bit!

**Comment by pnrobinson** [comment]
Extend parser for MGI_GenePheno.rpt to get the "|"-separated PMIDs before merging

**Comment by pnrobinson** [comment]
The "|" is working and now has a test
